types/xmlbuilder/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for xmlbuilder
 // Project: https://github.com/oozcitak/xmlbuilder-js
 // Definitions by: Wallymathieu <https://github.com/wallymathieu>
+//               : GaikwadPratik <https://github.com/GaikwadPratik>
```

Comment:
I think there are a couple of more people that have made changes to these type definitions.
I think @higuri , @johanblumenberg and @damonpam should end up here as well, but that could be a separate pull request.